src/routes/web.php

```diff
@@ -11,4 +11,6 @@
 |
 */
 
-Route::view('/{any?}', 'app');
+Route::view('/', 'app');
+
+Auth::routes();
```

Comment:
Just a guess, but this should be in `src/routes/api.php`. You should check with @fabio10000

Reply:
Fixed in 3212365